<launch>
	<node name="foo" pkg="vision_server" type="unit_test.py" />
</launch>
